server/pulp/server/async/app.py

```diff
@@ -135,8 +135,13 @@ def get_resource_manager_lock(name):
             lock.timestamp = now
             lock.save()
 
-            msg = _("Resource manager '%s' has acquired the resource manager lock") % name
-            _logger.info(msg)
+            msg = _("Resource manager '%s' has acquired the resource manager lock" % name)
```

Comment:
Oh this won't i8ln correctly. The string replacement has to happen on the response from `_()` not on the argument passed into `_()`.